core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/rabbitmq/check-version-debian.yml

```diff
@@ -0,0 +1,4 @@
+- name: RabbitMQ | Check Debian package version
+  command: dpkg-query --showformat='${Version}' --show rabbitmq-server
```

Comment:
Would this be possible to use here `package_facts` https://docs.ansible.com/ansible/latest/collections/ansible/builtin/package_facts_module.html ? :thinking: